test/indexing.jl

```diff
@@ -6,12 +6,12 @@ using Test, DataFrames
     df = DataFrame(a=1:3, b=4:6, c=7:9)
 
     @test df[1] == [1, 2, 3]
-    @test df[1] === eachcol(df, false)[1]
+    @test df[1] === eachcol(df)[1]
     @test df[1:2] == DataFrame(a=1:3, b=4:6)
-    @test eachcol(df[1:2], false)[1] === eachcol(df, false)[1]
+    @test eachcol(df[1:2])[1] === eachcol(df)[1] === eachcol(df)[1]
```

Comment:
```suggestion
    @test eachcol(df[1:2])[1] === eachcol(df)[1] === eachcol(df, true)[1]
```

Reply:
I will add it to the batch so that it is not pending, but this will fail (I will later fix it so that it does not fail)